utils/build/docker/nodejs/express4-typescript/app.ts

```diff
@@ -36,7 +36,8 @@ app.get('/healthcheck', (req: Request, res: Response) => {
     library: {
       language: 'nodejs',
       version: require('dd-trace/package.json').version,
-      libddwaf_version: require('@datadog/native-appsec/package.json').libddwaf_version
+      libddwaf_version: require('dd-trace/node_modules/@datadog/native-appsec/package.json').libddwaf_version,
+      appsec_event_rules_version: require('dd-trace/packages/dd-trace/src/appsec/recommended.json').metadata.rules_version
```

Comment:
It's unfortunately a little bit more complex. This code must get the version of the rule file actually used by appsec extension, not the version of the file shipped by default : 

This rules file can be a custom one, specified by `DD_APPSEC_RULES` env var. In that situation, the version returned here must be the version of this file (see ` utils/build/docker/overwrite_waf_rules.Dockerfile`).

@simon-id do you know if there is an easy solution to do so ?

Reply:
Oh is this a new requirement ? because previously it was like this:
```npm explore dd-trace -- cat packages/dd-trace/src/appsec/recommended.json | jq -r '.metadata.rules_version // "1.2.5"' > SYSTEM_TESTS_APPSEC_EVENT_RULES_VERSION```
which was only getting the default version, not a custom one.

```suggestion
      appsec_event_rules_version: require('dd-trace/packages/dd-trace/src/appsec/waf').wafManager?.rulesVersion
```

Reply:
That's the first layer. Then another logic that overwrite the value is executed here : https://github.com/DataDog/system-tests/blob/main/utils/build/build.sh#L234-L246

Reply:
well this new suggestion i'm proposing would get the version used by the waf at any time. but i don't think it would take RC changes into account

Reply:
That's perfect

Reply:
@simon-id `wafManager` is `null` when dd-trace is not initialized.

Reply:
Actually never mind it should be initialized at that point, so not sure why it's still `null` 🤔

Reply:
Looks like some tests are testing missing rules, so of course there is no version reported in that case.

@simon-id Any ideas to work around that?

Reply:
I think using `SYSTEM_TESTS_APPSEC_EVENT_RULES_VERSION` when present and otherwise falling back to the previous logic of checking `.metadata.rulesVersion` would be equivalent to how it worked before.

Reply:
Ended up keeping `wafManager.rulesVersion` since that would always be more accurate when present, and then falling back to `SYSTEM_TESTS_APPSEC_EVENT_RULES_VERSION` when it's not set.

Reply:
The above didn't work, so I tried to fallback to `require('dd-trace/packages/dd-trace/src/appsec/recommended.json').metadata.rules_version` only when the environment variable isn't set as it was before but at runtime, and it still doesn't work.

@simon-id Any idea what is the correct ordering of things here?

Reply:
wdym it doesn't work ?

Reply:
no idea about build.sh, but the js suggestion i made should definitely work no ?